### PR TITLE
feat: allow to filter props inherited from mui (#116)

### PR DESCRIPTION
**Pull request by Ajalle**

### CODE REVIEW

**Comment by Ajalle** [comment]
> Hi @Ajalle, I think we can globally hide controls in the `packages/data-explorer-ui/.storybook/preview.js` file by adding the following:
> 
> ```
>   controls: {
>     exclude: ["className", "ref", "sx"],
>     expanded: true,
>     matchers: {
>       color: /(background|color)$/i,
>       date: /Date$/,
>     },
>   },
> ```
> 
> Let's revert these changes and update the parameters in the `preview.js` file instead.
> 
> Would you mind also adding `ref` to the exclusion list for me too please. Thank you ⭐



> Hi @Ajalle, I think we can globally hide controls in the `packages/data-explorer-ui/.storybook/preview.js` file by adding the following:
> 
> ```
>   controls: {
>     exclude: ["className", "ref", "sx"],
>     expanded: true,
>     matchers: {
>       color: /(background|color)$/i,
>       date: /Date$/,
>     },
>   },
> ```
> 
> Let's revert these changes and update the parameters in the `preview.js` file instead.
> 
> Would you mind also adding `ref` to the exclusion list for me too please. Thank you ⭐

But this way we wouldn't allow `className` or `ref`, even if it is our custom prop, not just inherited from MUI. 
From our last conversation I understood that we want to keep the power to control it with ourselves. 
We also have many places where `className` defined as custom prop, so for me this top level exclusion sound like a hack to just hide those definitions instead of cleaning them.

**Comment by frano-m** [comment]
This is standard Storybook API. We can, if we want to, include these props back into a story, should we want to.
See https://storybook.js.org/docs/react/writing-stories/parameters#rules-of-parameter-inheritance.
This was something we did discuss.
The className prop is not necessarily a Mui related prop. We use it for [styling a component](https://emotion.sh/docs/styled#styling-any-component).

File: packages/data-explorer-ui/.storybook/main.ts
Comment (frano-m) [inline packages/data-explorer-ui/.storybook/main.ts:1]: Hi @Ajalle, would you mind completely reverting `main.ts` so there are no changes as part of this issue? Thank you!

**Review by frano-m — CHANGES_REQUESTED** [review]
Hi @Ajalle let's completely revert the `main.ts` file so there are no changes associated with this  issue! Thank you.

**Review by frano-m — APPROVED** [review]
Awesome @Ajalle ⭐ and thank you

**Pull request merged**